content-assignment-3.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import styles from "./index.module.css";
+const toggleData = () => {
+  let counter = 0;
+  const handleBoxClick = event => {
+    if (counter < 9 && !event.target.innerText) {
+      counter += 1;
+      event.target.innerText = counter;
+    }
+  };
+
+  return (
+    <div className={styles.content}>
+      <div className={styles.table}>
+        <div className={styles.box} onClick={handleBoxClick} />
+        <div className={styles.box} onClick={handleBoxClick} />
+        <div className={styles.box} onClick={handleBoxClick} />
+      </div>
+      <div className={styles.table}>
+        <div className={styles.box} onClick={handleBoxClick} />
+        <div className={styles.box} onClick={handleBoxClick} />
+        <div className={styles.box} onClick={handleBoxClick} />
+      </div>
+      <div className={styles.table}>
+        <div className={styles.box} onClick={handleBoxClick} />
+        <div className={styles.box} onClick={handleBoxClick} />
+        <div className={styles.box} onClick={handleBoxClick} />
+      </div>
+    </div>
+  );
+};
+
+export default toggleData;
+
+// Css
```

Comment:
Add it in a separate file called `index.module.css`.

Reply:
made the required changes